Gaijinizer patch/Map061.txt

```diff
@@ -198,96 +217,103 @@ E-003を投下する前にあの男が死ねば…
 少なくとも、E-003を解き放つ必要はなくなる。
 そういうこったろ？
 > CONTEXT: Map061/events/5/pages/0/123/Dialogue/other2
-If the man dies before dropping E-003 ...
-At least, there is no need to release E-003.
-Is that what happened?#MTLed
+At the very least, if that man dies before E-003
+gets airdropped in, there'll be no need to release
+it, right?#Ron
 > END STRING
 
 > BEGIN STRING
 お前………。
 > CONTEXT: Map061/events/5/pages/0/128/Dialogue/other2
-you……….#MTLed
+You can't...#Ron
 > END STRING
 
 > BEGIN STRING
 分かってるよ。
 
 命令違反者が、最悪殺されるってことくらい…
 > CONTEXT: Map061/events/5/pages/0/130/Dialogue/other2
-I know. The order violator is about to be killed
-in the worst ...#MTLed
+I know.
+
+Worst case, anyone who goes against orders gets
+killed...#Ron
 > END STRING
 
 > BEGIN STRING
 だけど、俺はそれ以上に…
 
 自分の手で、死神みてぇなE-003を解き放ちたくねぇ…
 > CONTEXT: Map061/events/5/pages/0/134/Dialogue/other2
-But I am more than that...With my own hands, I
-want to unleash the evil god E-003 ...#MTLed
+But at the very least I...
+
+I don't want these hands of mine to be the ones to
+release that angel of death, E-003...#Ron
```

Comment:
Same here. In addition, you wouldn't know this from where you are in the story, but E-003 is the furthest thing from a divine angel you could think of, so it'd be best to describe her in different terms...

>_**I don't want these hands of mine to be the ones
to release that specter of death, E-003...**_

(Feel free to change "specter" to "god" if you think it doesn't fit here.)